MediaBrowser.XbmcMetadata/Parsers/SeriesNfoParser.cs

```diff
@@ -110,6 +112,29 @@ protected override void FetchDataFromXmlNode(XmlReader reader, MetadataResult<Se
                         break;
                     }
 
+                case "namedseason":
+                    {
+                        var parsed = int.TryParse(reader.GetAttribute("number"), NumberStyles.Integer, CultureInfo.InvariantCulture, out var seasonNumber);
+                        var name = reader.ReadElementContentAsString();
+
+                        if (!string.IsNullOrWhiteSpace(name) && parsed)
+                        {
+                            if (item.SeasonNames == null)
+                            {
+                                item.SeasonNames = new Dictionary<int, string>()
```

Comment:
Aren't the parentheses redundant?

Reply:
The outer if clause checks if the name is set and parsing the series number to an int was successfull, the inner if checks if the item already has a dictionary assigned, if it hasn't, one is created with the new K/V pair, if it has the K/V pair is added to the existing dict.

Or does the item automatically have a dictionary assigned because the property is not set as nullable in the class - that would render the check useless?

Reply:
When you use an initializer, the parentheses are redundant e.g.

`var myList = new List<int> { 1, 2, 3, 4 };`

Reply:
TBF, I still type them